pkg/label/label.go

```diff
@@ -9,10 +9,16 @@ const (
 	// LegacyClusterID is an old style label for ClusterID
 	LegacyClusterID = "clusterID"
 
+	// LegacyClusterKey is an old style label for cluster encryption key.
+	LegacyClusterKey = "clusterKey"
```

Comment:
Should there be something like `giantswarm.io/cluster-key` to bring this in line with our current k8s label style?

Reply:
Yes. See https://github.com/giantswarm/randomkeys/blob/573b66dfb3de656f1f28c1d8b39104bf901a5b9c/types.go#L6-L16.

Reply:
I think it should be something more generic like `cert-operator.giantswarm.io/cert-name`.

Reply:
> I think it should be something more generic like cert-operator.giantswarm.io/cert-name.

I assume this is just a structural example? Do you have any suggestions for actual name? If I understood correctly, this encryption key is used for Guest k8s API -> etcd encryption so that etcd data is encrypted at rest, but I'm really out of imagination for good name here. I'm also not fully sure if there should be any `xyz-operator` prefix for `giantswarm.io` because this value seems to be used in all providers (i.e. AWS, Azure and KVM) as it's put to k8scloudconfig template. Ideas?

Reply:
I created a PR here: https://github.com/giantswarm/fmt/pull/9